src/issue-activity.ts

```diff
@@ -19,12 +19,42 @@ import {
 } from "./start";
 
 export enum CommentType {
+  /**
+   * Review related item
+   */
   REVIEW = 0b1,
+  /**
+   * Issue related item
+   */
   ISSUE = 0b10,
+  /**
+   * User assigned to the {@link CommentType.ISSUE} or {@link CommentType.REVIEW}
+   */
   ASSIGNEE = 0b100,
+  /**
+   * The author of the {@link CommentType.ISSUE} or {@link CommentType.REVIEW}
+   */
   ISSUER = 0b1000,
+  /**
+   * A user that is part of the organization or owner of the repo
+   */
   COLLABORATOR = 0b10000,
+  /**
+   * A user that is NOT part of the organization nor owner of the repo
+   */
   CONTRIBUTOR = 0b100000,
+  /**
+   * A user comment action on a {@link CommentType.ISSUE} or {@link CommentType.REVIEW}
```

Comment:
What is this @link syntax

Reply:
Simply makes the docs linking the code within the comment, for convenience, and looks like this
<img width="598" alt="image" src="https://github.com/ubiquibot/conversation-rewards/assets/9807008/76a60610-f3e4-4886-a9bd-c34a4c67a198">